src/components/AssetSelect.jsx

```diff
@@ -49,6 +50,14 @@ const SelectAsset = () => {
         );
     };
 
+    const assets = () => {
+        const names = Object.keys(pairs).map((pair) => {
+            return pair.split("/")[0];
+        });
+        names.push(LN);
+        return names;
+    };
```

Comment:
Why is this a function? It does only ever need to be evaluated once. Configs don't do reactive changes.